src/test/java/cz/inventi/kpj/kpjtesting/KpjTestingApplicationTests.java

```diff
@@ -15,35 +25,47 @@
  * 3.2. Assert the {@link KpjTesting#printEcho(String)} returns "KPJ echo"<br/>
  */
 
+@SpringBootTest
+@ExtendWith(MockitoExtension.class)
 class KpjTestingApplicationTests {
 
-
+  @MockBean
   HelloWorldService helloWorldService;
 
   @Autowired
   KpjTesting kpjTesting;
 
-  void testHelloWorld() {
+
+ @Test
+ void testHelloWorld() {
     // given
     // Setup mock
+   helloWorldService = Mockito.mock(HelloWorldService.class);
+   kpjTesting = new KpjTesting(helloWorldService);
```

Comment:
You just override fields `helloWorldService` and `kpjTesting` . Both of these fields should be as Spring beans. 
`kpjTesting` is created by Spring and you should just provide a mock bean for `HelloWorldService`. This test is for trying Spring integration for Mockito.
```suggestion
```